app/services/articles/feed.rb

```diff
@@ -38,15 +38,34 @@ def default_home_feed_and_featured_story(user_signed_in: false)
           limited_column_select.order("published_at DESC").limit(rand(15..80))
         hot_stories = hot_stories.to_a + new_stories.to_a
       end
-      hot_stories = rank_and_sort_articles(hot_stories) if @user
+      hot_stories = rank_and_sort_articles(hot_stories) if @user && ranking
       [featured_story, hot_stories]
     end
 
+    def default_home_feed_without_ranking
+      _featured_story, stories = default_home_feed_and_featured_story(user_signed_in: true, ranking: false)
+      stories
+    end
+
     def default_home_feed(user_signed_in: false)
-      _featured_story, stories = default_home_feed_and_featured_story(user_signed_in: user_signed_in)
+      _featured_story, stories = default_home_feed_and_featured_story(user_signed_in: user_signed_in, ranking: true)
       stories
     end
 
+    def ab_test_user_signed_in_feed(test_variant)
```

Comment:
I don't love this test being in this class, but it works right now. Perhaps the AB test switch should move to the controller level in the future?